### ISSUE TITLE
(STDERR) The spacjamio:chai package has been renamed to practicalmeteor:chai. Please use the new package name instead.

### ISSUE THREAD

**Issue by sferoze** [mad-eye/meteor-mocha-web#174]
When I add this package it automatically adds 'spacejamio:chai' which causes and error message. 

```
W20150601-14:04:10.108(-4)? (STDERR) The spacjamio:chai package has been renamed to practicalmeteor:chai. Please use the new package name instead.
```

When I remove mike:mocha the error stops. See full logs:

```
Ferozes-MacBook-Pro:Stridepost Nearpoint$ meteor add mike:mocha
                                              
Changes to your project's package version selections:
                                              
amplify                           added, version 1.0.0
mike:mocha                        added, version 0.6.0
mike:mocha-core                   added, version 0.1.0
package-version-parser            added, version 3.0.3
practicalmeteor:chai              added, version 2.1.0_1
practicalmeteor:loglevel          added, version 1.1.0_2
sanjo:long-running-child-process  added, version 1.0.3
sanjo:meteor-files-helpers        added, version 1.1.0_6
sanjo:meteor-version              added, version 1.0.0
spacejamio:chai                   added, version 1.9.2_3
velocity:chokidar                 added, version 0.12.6_1
velocity:core                     added, version 0.6.1
velocity:html-reporter            added, version 0.5.3
velocity:meteor-internals         added, version 1.1.0_7
velocity:shim                     added, version 0.1.0

                                              
mike:mocha: Run mocha tests in the browser    
Ferozes-MacBook-Pro:Stridepost Nearpoint$ meteor
[[[[[ ~/NewFoundry/Stridepost ]]]]]           

=> Started proxy.                             
=> Started MongoDB.                           
W20150601-14:04:10.108(-4)? (STDERR) The spacjamio:chai package has been renamed to practicalmeteor:chai. Please use the new package name instead.
```

**Comment by mhwheeler** [comment]
I investigated if removing `spacejamio:chai@1.9.2_3` from `mike:mocha/.versions` but it did not seem sufficient to make the error message go away. Something else involved seems to be creating the dependency?

**Comment by sferoze** [comment]
Hmm when I remove the package the error goes away. I even tried creating a new Meteor project and adding mike:mocha and I got the same error. I am pretty sure the error is associated with this package.

**Comment by mhwheeler** [comment]
I meant a dependency of `mike:mocha` must be creating the dependency, not the package directly. I was able to reproduce.

**Comment by isaaclyman** [comment]
I'm getting the same error, and have verified what previous comments have said.

**Comment by kidsil** [comment]
Getting the same error as well, 
mike:mocha@0.6.0
mike:mocha-core@0.1.0

**Comment by sungwoncho** [comment]
+1

**Comment by jacobdr** [comment]
+1 I just installed this package and am running into the same issues

**Comment by jacobdr** [comment]
velocity:core@0.6.0-rc.5 has as its own .versions dependency the spacejamio:chai package, so it seems like it will also be necessary to bump that version number to the current release (velocity:core@0.7.1) in order to rid ourselves of this dependency scourge. This is in addition to the pull request #179 that @kidsil is trying to pull. 

However, upgrading the velocity:core version number and the removing the spacejamio:chai package from .versions were not sufficient on my machine to prevent `meteor add` from still pulling down the spacejamio:chai package.... Frustrating. 

After about an hour of dependency snooping I wish that the [version sovler](https://www.meteor.com/version-solver) would provide us some better explanation of why certain zombie dependencies ultimately get added to the package....

**Comment by christiangenco** [comment]
:+1:

**Comment by rissem** [comment]
Thought the fix for this would be pretty simple, but it's not..

practicalmeteor:loglevel which velocity depends on indirectly depends on an old version of chai. I filed an issue w/ @practicalmeteor. Once that's resolved we should be able to change the dependencies all the way up the tree. 

More info here
https://github.com/practicalmeteor/meteor-loglevel/issues/4

By the way, does anyone know a nice way to visualize the dependency graph of a Meteor project? It seems like it would be super useful for these kinds of issues.

**Comment by rissem** [comment]
Fixed with https://github.com/mad-eye/meteor-mocha-web/commit/54a78d850cc49a817106719fa4757539cb2f265b

Just released the fix as 0.6.3
